collect_app/src/main/java/org/odk/collect/android/preferences/SubmissionsSharedPreferences.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.odk.collect.android.preferences;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import org.odk.collect.android.application.Collect;
+
+import java.util.Map;
+
+public class SubmissionsSharedPreferences {
```

Comment:
@grzesiek2010 Hey! What's the rationale for creating this class to separate the `SharedPreferences` work from the `SubmissionManager`?

Reply:
I wanted that to work like other (General and Admin) preferences to keep the implementation consistent. Don't you like it?